lib/oids.js

```diff
@@ -35,6 +35,8 @@ _IN('1.2.840.113549.1.1.11', 'sha256WithRSAEncryption');
 _IN('1.2.840.113549.1.1.12', 'sha384WithRSAEncryption');
 _IN('1.2.840.113549.1.1.13', 'sha512WithRSAEncryption');
 
+_IN('1.2.840.10040.4.3', 'dsa-wirh-sha1');
```

Comment:
Typo "wirh" => "with".

Wrt why we name things this way ... It would be nice to have compatibility with node's `crypto` lib, but there are lots of other crypto libs too -- and unless they are all using the same string, we'd need to alias for each one. So the general guideline here is to use what gets registered in the OID registry.